ansible/roles/set-repositories/tasks/main.yml

```diff
@@ -18,3 +18,7 @@
     - name: Configure Repository File
       import_tasks: ./file-repos.yml
       when: repo_method == "file"
+
+    - name: Configure Repository File
+      import_tasks: ./satellite-cert.yml
```

Comment:
same, `include_tasks`

Reply:
Why don't you update the satellite-repo.yml  and add a condition with `satellite_cert` there ?
`satellite-cert.yml` and `satellite-repo.yml` look very similar

Reply:
These have been merged into `satellite-repo.yml`